test/lib/client.test.js

```diff
@@ -41,10 +41,23 @@ function* waitForQueue(queue, condition) {
 
 describe('Node AMQP Bus Client', function testBus() {
   describe('#createBusClient', () => {
+    let sandbox;
+
+    beforeEach(() => {
+      sandbox = sinon.sandbox.create();
```

Comment:
this method is deprecated. Use `sinon.sinon.createSandbox()` instead

Reply:
Thanks for the remark, done.